apps/discovery_api/lib/discovery_api_web/controllers/multiple_metadata_controller.ex

```diff
@@ -101,6 +103,29 @@ defmodule DiscoveryApiWeb.MultipleMetadataController do
     )
   end
 
+  defp build_search_opts(params) do
+    query = Map.get(params, "query", "")
+    facets = Map.get(params, "facets", %{})
+    api_accessible = parse_api_accessible(params)
+
+    case validate_facets(facets) do
+      {:ok, filter_facets} ->
+        opts =
+          [
+            query: query,
+            api_accessible: api_accessible,
+            keywords: Map.get(filter_facets, :keywords),
+            org_title: Map.get(filter_facets, :organization, []) |> List.first()
```

Comment:
I hope someday that this `org_title` name will flow downstream into the UI (in vX.0 of the API) someday so we're no longer filtering on the hyper ambiguous `organization`

Reply:
Might be a good enhancement once we get it wired up to the ui